api/provider/aws/load_balancer_list.go

```diff
@@ -0,0 +1,76 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (e *LoadBalancerProvider) List() ([]models.LoadBalancerSummary, error) {
+	loadBalancerNames, err := e.listLoadBalancerNames()
+	if err != nil {
+		return nil, err
+	}
+
+	summaries := make([]models.LoadBalancerSummary, len(loadBalancerNames))
+	for i, loadBalancerName := range loadBalancerNames {
+		loadBalancerID := delLayer0Prefix(e.Config.Instance(), loadBalancerName)
+		summary := models.LoadBalancerSummary{
+			LoadBalancerID: loadBalancerID,
+		}
+
+		summaries[i] = summary
+	}
+
+	if err := e.populateSummariesTags(summaries); err != nil {
+		return nil, err
+	}
+
+	return summaries, nil
+}
+
+func (e *LoadBalancerProvider) listLoadBalancerNames() ([]string, error) {
+	output, err := e.AWS.ELB.DescribeLoadBalancers(&elb.DescribeLoadBalancersInput{})
```

Comment:
Load balancers can return multiple pages of results.
http://docs.aws.amazon.com/sdk-for-go/api/service/elb/#DescribeLoadBalancersInput

Even though the default is 400 for the page size, I think either a comment here stating we aren't checking for multiple pages because the default is sufficiently high enough (if that is the case) or ensure we iterate through all the pages (possibly better option).